### PR TITLE
Roku docs

### PR DESCRIPTION
**Pull request by zzhaobraze**
# Pull Request/Issue Resolution
Adding roku docs to documentations.

**Description of Change:**
Initial Roku docs

### CODE REVIEW

**Comment by EmilyNecciai** [comment]
@Bucimis I'll add a do not merge label, but I'll review and we'll merge a few days before that release. :) Plenty of time!

**Comment by EmilyNecciai** [comment]
Note to self: 
- [ ] add 10kB sdk file size to docs.

**Comment by Bucimis** [comment]
@EmilyNecciai this is ready to be merged and deployed (ideally for tomorrow's major release)

(cc @dkim32)

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_user_ids.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| Identification Context | Preservation Behavior |
+| ---------------------- | -------------------------- |
+| User __has not__ been previously identified | Anonymous history __is merged__ with user profile upon identification |
+| User __has been__ previously identified in-app or via API | Anonymous history __is not merged__ with user profile upon identification |
+
+#### Additional Notes and Best Practices
+
+Please note the following:
+
+- __If your app is used by multiple people, you can assign each user a unique identifier to track them.__
```
Comment (dkim32) [inline _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_user_ids.md]: minor - but i'd remove the bold styling here given that multiple points are highlighted.

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+Custom Attributes can also be unset using the following method:
+
+```
+m.Braze.unsetCustomAttribute("attributeName")
+```
+
+#### Setting a Custom Atttribute via the REST API
+
+You can also use our REST API to set user attributes. To do so refer to the [user API documentation][4].
```
Comment (dkim32) [inline _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md]: this link is broken.

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+#### Custom Attribute Value Limits
+
+Custom attribute values have a maximum length of 255 characters.
+
+### Managing Notification Subscription Statuses
```
Comment (dkim32) [inline _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md]: does this make sense to include push subscription type given that we don't have push to this platform to start?

File: _docs/_developer_guide/platform_integration_guides/roku/initial_sdk_setup.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+nav_title: Initial SDK Setup
+platform: Roku
+page_order: 0
+---
+
+# Initial SDK Integration
+
+Installing the Braze Roku SDK will provide you with basic analytics, segmentation, and engagement functionality.
```
[the remark this answers is not in the document]
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/roku/initial_sdk_setup.md]: good call, removed

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_user_ids.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_user_ids.md]: good call - this was pattern matching. removed

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_user_ids.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+nav_title: Setting User IDs
+platform: Roku
+page_order: 0
+---
+## Setting User IDs
+
+User IDs should be set for each of your users. These should never change per user and be readily accessible when a user opens the app. A database ID or a hashed email address/username is usually a good reference to use. We __strongly recommend__ providing this identifier as it will allow you to:
+
+- Track your users across devices and platforms, improving the quality of your behavioral and demographic data.
```
[the remark this answers is not in the document]
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_user_ids.md]: fixed

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md (hunk 3 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md]: true, removed docs for the push side (although the method will still be there in the sdk)

File: _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md (hunk 2 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/roku/analytics/setting_custom_attributes.md]: it worked for me and seems to be identical to other links linking out to the rest api 🤔

**Pull request merged**